lib/api-gateway.ts

```diff
@@ -0,0 +1,162 @@
+import apigw = require('@aws-cdk/aws-apigateway');
+import { Metric, MetricOptions, ComparisonOperator, GraphWidget } from '@aws-cdk/aws-cloudwatch';
+import { Construct, Duration } from '@aws-cdk/core';
+import { IWatchful } from './api';
+
+export interface WatchApiGatewayRestApiOptions {
+  /**
+   * Alarm when 5XX errors reach this threshold over 5 minutes.
+   *
+   * @default - no alarm.
```

Comment:
Why no alarm by default. I think we should have an alarm set up by default for 500s

Reply:
Sure, making it default to `1`.